src/components/common/RegAddEditOrgPerson.vue

```diff
@@ -215,46 +137,27 @@
                   </v-card>
                 </div>
                 <div v-else-if="isProprietor">
-                  <p class="mt-6 mb-0">
-                    To add a registered B.C. business or corporation as the Proprietor, enter the name
-                    or incorporation number.
-                  </p>
-                  <p class="mt-6 mb-0">
-                    If you are adding a company that is not legally required to register in B.C. such as
-                    a bank or a railway, use the manual entry form. All other types of business cannot
-                    be the Proprietor.
+                  <p class="mb-0">
+                    Enter an existing B.C. business as the proprietor.
                   </p>
                 </div>
                 <div v-else-if="isPartner">
-                  <p class="mt-6 mb-0">
-                    To add a registered B.C. business or corporation as a partner, enter the name or
-                    incorporation number.
-                  </p>
-                  <p class="mt-6 mb-0">
-                    If you are adding a company that is not legally required to register in B.C. such as
-                    a bank or a railway, use the manual entry form. All other types of business cannot
-                    be a partner.
-                  </p>
-                  <HelpContactUs class="mt-6" />
-                </div>
-                <div v-else>
-                  <p class="mt-6 mb-0">
-                    To add a registered B.C. business or corporation, enter the name or incorporation
-                    number.
-                  </p>
-                  <p class="mt-6 mb-0">
-                    If you are adding a company that is not legally required to register in B.C. such as
-                    a bank or a railway, use the manual entry form.
+                  <p class="mb-0">
+                    Enter an existing B.C. business as a partner.
                   </p>
+
                 </div>
 
                 <BusinessLookup
                   :showErrors="enableRules"
                   :businessLookup="inProgressBusinessLookup"
                   :BusinessLookupServices="BusinessLookupServices"
+                  :label="businessLookupLabel"
                   @setBusiness="updateBusinessDetails($event)"
                   @undoBusiness="resetBusinessDetails()"
                 />
+
+                <HelpContactUs class="mt-0.25" />
```

Comment:
to meet the 30px space above and 40px space below requirement
![image](https://github.com/bcgov/business-create-ui/assets/144159934/c91e9c90-2a14-4c9f-a24c-188b2214ad94)

Reply:
Where did you find this classname?

Reply:
I find it from (https://vuetifyjs.com/en/styles/spacing/#how-it-works), I try different spacing to make sure it is 30px above and 40px below
![image](https://github.com/bcgov/business-create-ui/assets/144159934/7dd6ef82-cbcc-4b7f-bb42-359e59c4e529)
![image](https://github.com/bcgov/business-create-ui/assets/144159934/7c99e7e4-e6a9-4d3f-be7b-cfd49e9b2dcd)

Reply:
I think that classname doesn't exist.

Look, it doesn't have a definition in the DOM:

![image](https://github.com/bcgov/business-create-ui/assets/10002889/47302d99-2248-4ab8-8d0b-a298ca55aead)

In fact, there isn't a Vuetify helper class for "30px". We either use 30px or 32px (factors of 4). Ask Yui what to do here.

(Example: mt-3 means 3/4 rem = 12 px.)

Reply:
LGTM